clang/test/Driver/linker-wrapper-sycl-win.cpp

```diff
@@ -90,7 +90,7 @@
 // CHK-CMDS-AOT-NV-NEXT: "{{.*}}llvm-link.exe" -only-needed [[FIRSTLLVMLINKOUT]].bc {{.*}}.bc -o [[SECONDLLVMLINKOUT:.*]].bc --suppress-warnings
 // CHK-CMDS-AOT-NV-NEXT: "{{.*}}sycl-post-link.exe"{{.*}} SYCL_POST_LINK_OPTIONS -o [[SYCLPOSTLINKOUT:.*]].table [[SECONDLLVMLINKOUT]].bc
 // CHK-CMDS-AOT-NV-NEXT: "{{.*}}clang.exe"{{.*}} -o [[CLANGOUT:.*]] --target=nvptx64-nvidia-cuda -march={{.*}}
-// CHK-CMDS-AOT-NV-NEXT: "{{.*}}ptxas"{{.*}} --output-file [[PTXASOUT:.*]] [[CLANGOUT]]
+// CHK-CMDS-AOT-NV-NEXT: "{{.*}}ptxas.exe"{{.*}} --output-file [[PTXASOUT:.*]] [[CLANGOUT]]
```

Comment:
I don't think it will work. I see that passes on some machines. 
If you want to match both spellings, it's better to make following change:
```suggestion
// CHK-CMDS-AOT-NV-NEXT: "{{.*}}ptxas{{.*}} --output-file [[PTXASOUT:.*]] [[CLANGOUT]]
```

Reply:
ah yeah good idea

Reply:
there is no `REQUIRES` that would prevent it from running without cuda installed, so probably clang's execution finding failed so it didn't add the extension or something.

Reply:
CUDA wasn't installed, so I can't imagine it's 2. I'll test it on a machine without CUDA.

Reply:
@bader It was indeed passing, and the passing output was this


```
 "ptxas" -m64 -O2 --gpu-name sm_50 --output-file C:\Users\nsarnie\AppData\Local\Temp\3\a.out-4fa681.cubin C:\Users\nsarnie\AppData\Local\Temp\3\a.out.nvptx64.sm_50-f7a776.img
"fatbinary" -64 --create C:\Users\nsarnie\AppData\Local\Temp\3\a.out-03c38f.fatbin --image=profile=compute_50,file=C:\Users\nsarnie\AppData\Local\Temp\3\a.out.nvptx64.sm_50-f7a776.img --image=profile=sm_50,file=C:\Users\nsarnie\AppData\Local\Temp\3\a.out-4fa681.cubin
```

so it looks like it generates a call with no `.exe` or path when it can't find it

Reply:
Here is why: https://github.com/intel/llvm/blob/sycl/clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp#L329-L330

Reply:
makes sense, thanks for investigating